Automattic/simplenote-android
# Remove sustainer UI

This PR removes the sustainer UI and related in-app-purchase code. I've opted to just remove everything, but if we think we might want to use the banners for something else in the future we could keep them around instead...

### Test
* Launch the app, logged out.
* Sign in to an account with or without sustainer associated with it.
* Open the drawer, you should not see the sustainer banner.
* Open settings, you should not see the sustainer banner.
* Verify other views in the app work as expected.

### Release
These changes do not require release notes.
